components/collection/drop/modal/DropConfirmModal.vue

```diff
@@ -34,6 +36,8 @@ import {
   useCountDown,
 } from '@/components/collection/unlockable/utils/useCountDown'
 
+type ModalStep = 'email' | 'claiming' | 'succeded'
```

Comment:
the code will have less "free roaming" strings around if you would used an enum...

Reply:
fixed